drivers/hmis/app/graphql/schema.graphql

```diff
@@ -4022,6 +4022,11 @@ type FormItem {
   text: String
   type: ItemType!
 
+  """
+  Component variant to use for display
+  """
+  variant: Variant
```

Comment:
@martha @ttoomey I'm wondering about the difference between `variant` and `component` on the Form Item?

Reply:
Good point, maybe they are redundant. My own understanding of the`component` prop is probably incorrect; I imagined it was to describe behavioral/functional differences (checkbox vs radio) rather then describing cosmetics. However I totally buy that distinction doesn't matter and consolidating around a single property is a better way to go.

Reply:
I see. The `component` prop should be 100% cosmetic. I can see the confusion with radio buttons, but it doesn't actually work that way. The `repeats: true` is the deciding factor on whether something ends up being a single-select or multi-select input.

Reply:
just to confirm, it sounds like I should change this back to using `component`, and remove `variant`?

Reply:
Sounds like it! Sorry for the confusion and thanks for clarifying @gigxz

Reply:
Yes! I am sorry for the delay! This looks great, I am going to re-target to release-104 and merge. Thanks both 🖊️ 👍